packages/lesswrong/server/search/algoliaDocuments.d.ts

```diff
@@ -19,7 +19,13 @@ interface AlgoliaComment {
   postId?: string,
   postTitle?: string,
   postSlug?: string,
+  postSequenceId?: string,
```

Comment:
I don't think this is being used anywhere else?

Reply:
it's used in `postGetPageUrl`
```
// Get URL of a post page.
export const postGetPageUrl = function(post: PostsMinimumForGetPageUrl, isAbsolute=false, sequenceId:string|null=null): string {
  const prefix = isAbsolute ? getSiteUrl().slice(0,-1) : '';

  // LESSWRONG – included event and group post urls
  if (sequenceId) {
    return `${prefix}/s/${sequenceId}/p/${post._id}`;
  } else if (post.isEvent) {
    return `${prefix}/events/${post._id}/${post.slug}`;
  } else if (post.groupId) {
    return `${prefix}/g/${post.groupId}/p/${post._id}/`;
  }
  return `${prefix}/posts/${post._id}/${post.slug}`;
};
```

Reply:
But `CommentsSearchHit` doesn't pass in `postSequenceId` when it calls `postGetPageUrl()`? Also we don't populate `postSequenceId` in Algolia so it would always be null anyway?